packages/react/src/QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem.tsx

```diff
@@ -63,15 +64,16 @@ export function QuestionnaireFormItem(props: QuestionnaireFormItemProps): JSX.El
   }
 
   const initial = item.initial && item.initial.length > 0 ? item.initial[0] : undefined;
-
   switch (type) {
     case QuestionnaireItemType.boolean:
       return (
         <CheckboxFormSection key={props.item.linkId} title={props.item.text} htmlFor={props.item.linkId}>
           <Checkbox
             id={props.item.linkId}
             name={props.item.linkId}
-            defaultChecked={initial?.valueBoolean}
+            defaultChecked={
+              initial?.valueBoolean ?? getDefaultAnswer(props.responses, item, index, props.groupSequence)
```

Comment:
Should `getDefaultAnswer` be before `initial?.valueBoolean`?  

If a user enters an answer, that should take precedence, right?